WooCommerce/Classes/ViewRelated/Orders/Order Creation/ProductsSection/ProductDiscountView.swift

```diff
@@ -2,45 +2,30 @@ import SwiftUI
 import Yosemite
 
 struct ProductDiscountView: View {
-    private let imageURL: URL?
-    private let name: String
-    private let stockLabel: String
-    private let productRowViewModel: ProductRowViewModel
-    private let priceSummaryViewModel: CollapsibleProductCardPriceSummaryViewModel
+    private var viewModel: ProductDiscountViewModel
```

Comment:
nit: can this be a `let`?
```suggestion
    private let viewModel: ProductDiscountViewModel
```

Reply:
Good call, yes! I'd made it a `var` when trying a different approach with `discountDetailsViewModel` but this way it can be a `let`.